DFC.Api.Content/appsettings.json

```diff
@@ -0,0 +1,39 @@
+{
+  "Neo4j": {
+    "Endpoints": [
+      {
+        "Name": "instance_0",
+        "Uri": "__Neo4jUrl__",
+        "Username": "__Neo4jUser__",
+        "Password": "__Neo4jPassword__",
+        "Enabled": "__Enabled__"
+      },
+      {
+        "Name": "instance_1",
+        "Uri": "__Neo4jUrl__",
```

Comment:
Should probably just set these to blank for now, except Enabled, which need to be hardcoded to false.